.github/CODEOWNERS

```diff
@@ -617,3 +617,33 @@ utils/bazel/llvm-project-overlay/libc/** @llvm/pr-subscribers-libc
 /clang/test/CodeGen/WebAssembly/ @llvm/pr-subscribers-webassembly
 /clang/test/Sema/*wasm* @llvm/pr-subscribers-webassembly
 /clang/test/SemaCXX/*wasm* @llvm/pr-subscribers-webassembly
+
+# llvm-binary-utilities
+/llvm/docs/CommandGuide/llvm-* @llvm/pr-subscribers-llvm-binary-utilities
```

Comment:
Seems that https://github.com/orgs/llvm/teams/pr-subscribers-llvm-binary-utilities is present?

Reply:
This looks like the issue we ran into for lldb too. Mehdi explained it here: https://discourse.llvm.org/t/input-needed-teams-for-pull-request-subscriptions/73116/89

Reply:
I know, but I got a warning when the team is actually present:)